packages/server-wallet/src/onchain-service/types.ts

```diff
@@ -0,0 +1,156 @@
+import {Address, Bytes32} from '@statechannels/client-api-schema';
```

Comment:
Feedback for this file.

I came to this file to try and see what is the interface that the wallet will use. But this file contains a mix of internal types (ie. implementation details) as well as external interfaces.

It might still make sense for this file to contain both of these types, so that all the types are imported from the same file. If that's the case, perhaps they could be organized (eg. with some comment headers) so that the external interface is clear.
```
// BEGIN EXTERNAL INTERFACE
 export type OnchainServiceInterface = { ... }
...
export type TransactionSubmissionStoreInterface = { ... } // Is this an external interface?
// END EXTERNAL INTERFACE

// BEGIN INTERNAL TYPES

// END INTERNAL TYPES
```

I might prefer two separate files, maybe `types/internal.ts` and `types/external.ts`